Makefile

```diff
@@ -109,7 +109,7 @@ test-advanced:
 
 test-behat:
 	bin/tools sh -c "APP_DEBUG=1 APP_ENV=test bin/post-install-dev.sh"
-	bin/tools sh -c "APP_DEBUG=0 APP_ENV=test vendor/bin/behat --format=progress"
+	bin/tools vendor/bin/behat --format=progress
```

Comment:
You must also remove all occurrences hardcoded in the project:
- Dockerfile
- docker-compose.yml
- .github/workflows/ci.yml

Otherwise, the Kernel will only load the .env file based on the system environment variable (`dev` according to docker-compose.yml), won't load the .env.test file and will use the dev/prod database instead of the test database.

Reply:
Done in bootstrap.php